cabal-install/src/Distribution/Deprecated/ParseUtils.hs

```diff
@@ -128,11 +128,6 @@ instance Monad ParseResult where
   ParseOk ws x >>= f = case f x of
```

Comment:
Same here (the disable directive is several lines before the diff hunk).

Reply:
I bundled it with the change for the above & used --amend to fold it into the last commit.